server/src/controllers/temp-outfitter.es6

```diff
@@ -491,45 +501,48 @@ tempOutfitter.update = (req, res) => {
     where: {
       app_control_number: req.params.id
     }
-  }).then(app => {
-    if (app) {
-      app.status = req.body.status;
-      app.applicantMessage = req.body.applicantMessage;
-      if (app.status === 'Accepted') {
-        tempOutfitter
-          .acceptApplication(app)
-          .then(response => {
-            app.controlNumber = response.controlNumber;
-            app
-              .save()
-              .then(() => {
-                res.status(200).json(translateFromDatabaseToClient(app));
-              })
-              .catch(error => {
-                res.status(500).json(error);
-              });
-          })
-          .catch(error => {
-            res.status(500).json(error);
-          });
-      } else {
-        app
-          .save()
-          .then(() => {
-            if (app.status === 'Returned') {
-              //TODO: remove conditional if we want to send emails to applications with Hold status
+  })
+    .then(app => {
```

Comment:
This section looks like it has a high cyclomatic complexity. Is there any way to break this promise chain out a bit to reduce it?